bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/i18n/shelly_de.properties

```diff
@@ -370,17 +462,19 @@ channel-type.shelly.timerAutoOff.description = Wenn das Relais eingeschaltet wir
 channel-type.shelly.timerActive.label = Timer aktiv
 channel-type.shelly.timerActive.description = ON: Auto-On/Off Timer ist aktiv
 channel-type.shelly.temperature1.label = Temperatur 1
-channel-type.shelly.temperature1.description = Temperatur des externen Sensors #1
+channel-type.shelly.temperature1.description = Temperatur des externen Sensors 1
 channel-type.shelly.temperature2.label = Temperatur 2
-channel-type.shelly.temperature2.description = Temperatur des externen Sensors #2
+channel-type.shelly.temperature2.description = Temperatur des externen Sensors 2
 channel-type.shelly.temperature3.label = Temperatur 3
-channel-type.shelly.temperature3.description = Temperatur des externen Sensors #3
+channel-type.shelly.temperature3.description = Temperatur des externen Sensors 3
 channel-type.shelly.humidity.label = Luftfeuchtigkeit
 channel-type.shelly.humidity.description = Relative Luftfeuchtigkeit (0..100%)
 channel-type.shelly.rollerShutter.label = Steuerung (0=offen, 100=geschlossen)
 channel-type.shelly.rollerShutter.description = Steuerung f�r den Rollladen: UP, DOWN, STOP, Position (0=offen, 100=geschlossen)
 channel-type.shelly.rollerPosition.label = Position (100=offen, 0=zu)
 channel-type.shelly.rollerPosition.description = Invertierte Position des Rollladen: 100=offen, 0=zu
+channel-type.shelly.rollerFavorite.label = Positionsfavorit
```

Comment:
"Bevorzugte Position" oder "Voreingestellte Position"?

Reply:
that's not what it mean, you could setup 4 favorites in the device and this channel allows the selection
I want to refer to "favorite", because that's the language used by the App

Any other idea?